frontend/src/theme/components/Tooltip.ts

```diff
@@ -0,0 +1,26 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export const TOOLTIP_THEME_KEY = 'CustomTooltip'
+export const CustomTooltip: ComponentMultiStyleConfig = {
+  parts: ['tooltip', 'wrapper'],
```

Comment:
I think we can just override the original tooltip styling if we don't actually need the special mobile handling code